### DISCUSSION TITLE
Questions around the licensing JSON element

### DISCUSSION THREAD

**Discussion by ghost** [wgu-opensource/osmt#73] [General]
Each skill can have its own license according to the schema ([https://purl.org/ctdlasn/terms/license](https://purl.org/ctdlasn/terms/license)).
 
Let’s say that job description 1 has 10 skills assigned to it (not uncommon), that are contributed by 10 different sources. There would then be 10 different licenses the end user would have to digitally accept to use the skills content.
 
How will that be facilitated, tracked, etc.?

**Comment by arblackwell1** [comment]
@ottonomy Can you help answer Dorian's question on licensing RSDs?

**Comment by AaronHanich** [comment]
The CDTLASN does define a license to be attached to an RSD, however OMST does not currently implement that.  I'm not sure myself on the process of licensing a Skill and not sure what type of storage for that information would be required, ie whether they are externally licensed that we'll integrate with or if it's a portable license we can store alongside the Skill.  Either way it will need to be a new feature in OSMT
